setup.cfg

```diff
@@ -1,2 +1,4 @@
 [flake8]
-max-line-length = 88
\ No newline at end of file
+max-line-length = 88
+exclude =
+    tests/*
```

Comment:
what's the motivation behind not linting tests?

Reply:
We should lint tests, I just had so many errors I originally decided to opt-out, but I can fix them instead 🙂

Reply:
Added back in now 👍